commands/Tools/contact.js

```diff
@@ -0,0 +1,71 @@
+exports.init = async (client) => {
```

Comment:
To keep consistency with the rest of pieces, please place exports.init below exports.conf and exports.help (just style)